aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -478,6 +478,26 @@ The handler settings can also be provided via environment variables:
 
 See [Configuration with web.config](#configuration-with-webconfig) for an example of the `aspNetCore` element in the *web.config* file.
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Modifying the stack size
+
+You can configure the default managed stack size using the `stackSize` setting. This value is in **bytes**. The default size is `1048576` bytes (1 MB).
```

Comment:
Use imperative mood to avoid 2nd person in most cases. It's ok to just order the reader to do something.

```suggestion
Configure the managed stack size using the `stackSize` setting in bytes. The default size is `1048576` bytes (1 MB).
```